libpod.conf

```diff
@@ -146,6 +146,23 @@ crun = [
 	    "/usr/local/bin/crun",
 ]
 
+# Kata Containers is an OCI runtime, where containers are run inside lightweight
+# Virtual Machines (VMs). Kata provides additional isolation towards the host,
+# minimizing the host attack surface and mitigating the consequences of
+# containers breakout.
+# Please notes that Kata does not support rootless podman yet, but we can leave
+# the paths below blank to let them be discovered by the $PATH environment
+# variable.
+
+# Kata Containers with the default configured VMM
+kata-runtime = []
+
+# Kata Containers with the QEMU VMM
+kata-qemu = []
```

Comment:
Shouldn't you have the default paths in here?

Reply:
They will be discovered by the `$PATH` env variable, should I add them nevertheless?

Reply:
The other OCI runtimes paths are mentioned.  If Fedora decides to install kata in a non standard path, /usr/lib/libexec/kata/kata or something like that, then we would want these paths initialized for the distribution.  I am not sure all distros want to support the kata executable directly or rather as a library/executable for other packages like CRI-O, Podman ...